.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
+
+on: workflow_dispatch
+
+env:
+  FOUNDRY_PROFILE: ci
```

Comment:
I haven't seen this in our other repository. I did quick research and it seems unnecessary?

Reply:
It is unnecessary. This workflow is automatically created on `forge init`.